.ci/scripts/test-GC-manifest.sh

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+readonly SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly CI_DIR="$( cd "${SCRIPTS_DIR}/.." && pwd )"
+readonly LIB_DIR="$( cd "${GOPATH}/src/github.com/kyma-project/test-infra/prow/scripts/lib" && pwd )"
+readonly CLUSTER_DIR="$( cd "${CI_DIR}/cluster" && pwd )"
+readonly RESOURCES_DIR="$( cd "${CLUSTER_DIR}/resources" && pwd )"
+readonly CLUSTER_CONFIG="${CLUSTER_DIR}/cluster.yaml"
+readonly KUBERNETES_VERSION="${KUBERNETES_VERSION:-"v1.15.3"}"
+readonly ARTIFACTS_DIR="${ARTIFACTS:-"${CI_DIR}/in"}"
+readonly INSTALLATIONTIMEOUT=1000 #in this case it mean 20 minutes
+readonly CLUSTER_NAME="${CLUSTER_NAME:-"kyma"}"
+readonly NAMESPACE="${NAMESPACE:-"default"}"
+readonly SERVICE_ACCOUNT="${SERVICE_ACCOUNT:-"kyma-serviceaccount"}"
+
+# shellcheck disable=SC1090
+source "${LIB_DIR}/log.sh"
+# shellcheck disable=SC1090
+source "${LIB_DIR}/junit.sh"
+# shellcheck disable=SC1090
+source "${LIB_DIR}/docker.sh"
+# shellcheck disable=SC1090
+source "${LIB_DIR}/kind.sh"
+# shellcheck disable=SC1090
+source "${LIB_DIR}/kubernetes.sh"
+
+ENSUREKUBECTL="false"
+START_DOCKER="false"
+TUNE_INOTIFY="false"
+DOMAIN=kyma.local
+
+function readFlags() {
+    while test $# -gt 0; do
+        case "$1" in
+            -h|--help)
+                shift
+                echo "Script that tests manifest for google-cloud marketplace"
+                echo " "
+                echo "Options:"
+                echo "  -h --help            Print usage."
+                echo "     --ensure-kubectl  Update kubectl to the same version as cluster."
+                echo "     --start-docker    Start the Docker Daemon."
+                echo "     --tune-inotify    Tune inotify instances and watches."
+                echo " "
+                echo "Environment variables:"
+                echo "  ARTIFACTS  If not set, all artifacts are stored in \`.ci/in\` directory"
+                exit 0
+                ;;
+            --ensure-kubectl)
+                shift
+                ENSUREKUBECTL="true"
+                ;;
+            --start-docker)
+                shift
+                START_DOCKER="true"
+                ;;
+            --tune-inotify)
+                shift
+                TUNE_INOTIFY="true"
+                ;;
+            *)
+                log::info "$1 is not a recognized flag, use --help flag for a list of avaiable options!"
+                return 1
+                ;;
+        esac
+    done
+
+    readonly ENSUREKUBECTL START_DOCKER TUNE_INOTIFY
+}
+
+function monitorInstallation(){
```

Comment:
```suggestion
function monitorInstallation() {
```

Also in other places